### PR TITLE
Create RumbleJack.json

### PR DESCRIPTION
**Pull request by RumbleJack**
Please merge me~

### CODE REVIEW

**Comment by RumbleJack** [comment]
How to use closes #7198 ,that's interesting...  thanks for @githubteacher

**Comment by RumbleJack** [comment]
someone help me.

File: _pins/RumbleJack.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: RumbleJack
+Your Latitude: 32.0617
```
Comment (brianamarie) [inline _pins/RumbleJack.json]: Hi @RumbleJack! Try changing "Your Latitude" to "latitude", and "Your Longitude" to "longitude".

Comment (RumbleJack) [inline _pins/RumbleJack.json]: Ok, thx!

Comment (RumbleJack) [inline _pins/RumbleJack.json]: Relief!

**Comment by brianamarie** [comment]
@RumbleJack  your code looks perfect! Please go ahead and click the big green button below to **Merge pull request**. In an effort to keep things in this repository clean, one of the trainers will merge this if it is still open in a few days.

**Pull request merged**